app/ui-react/packages/api/src/constants.ts

```diff
@@ -93,10 +93,6 @@ export enum FlowKind {
   DEFAULT = 'default',
 }
 
-export interface ITypedFlow extends Flow {
```

Comment:
I assume the new definitions already include the type?

Reply:
Yup, in the ugly diff: https://github.com/syndesisio/syndesis/pull/5740/files#diff-faacd889423a50a3458ee36fe93e65e1R291

Reply:
Cool thx!